### DISCUSSION TITLE
Cannot mark a notification as read on notification panel

### DISCUSSION THREAD

**Discussion by rklugman** [laravel/nova-issues#4273] [Q&A] [answered]
Laravel Version: 9.13.0
Nova Version: 4.6.0
PHP Version: 8.1
Database Driver & Version:
Operating System and Version: OSX 12.2.2
Browser type and version: Chrome 100.0.4896.75

### Description:

On the Notifications panel, clicking the X (mark as read) for a notification does not do anything.

<img width="211" alt="Screenshot 2022-05-18 at 15 32 37" src="https://user-images.githubusercontent.com/40756947/169051219-efc760f8-e091-40e6-be1a-edf3735fbff8.png">

**Comment by crynobone** [comment]
### When there unread notification

![image](https://user-images.githubusercontent.com/172966/169059496-f83dc63c-510f-4f87-99ec-c1dafb43a97b.png)
![image](https://user-images.githubusercontent.com/172966/169059514-9d9205d3-9bf7-42e2-bbde-f6834ed8bb7b.png)

### After clicking read

![image](https://user-images.githubusercontent.com/172966/169059574-23f12063-7772-4529-bb47-077dab281e25.png)
![image](https://user-images.githubusercontent.com/172966/169059598-277403fe-fe91-49f5-9e7b-443b14c47720.png)

**Comment by davidhemphill** [comment, marked as answer]
You appear to be using an older version of Nova. Since v4.3.0 you've been able to delete and mark notifications as read. Make sure you've updated Nova's assets using `php artisan nova:publish --force`.
